build.cake

```diff
@@ -131,25 +150,44 @@ Task("Test")
 	.IsDependentOn("Build")
 	.Does(() =>
 	{
-		NUnit3(UNIT_TEST_PATH);
+		NUnit3(BIN_DIR + UNIT_TEST_ASSEMBLY);
 	});
 
 //////////////////////////////////////////////////////////////////////
 // PACKAGE
 //////////////////////////////////////////////////////////////////////
 
-Task("Package")
+Task("PackageNuGet")
 	.IsDependentOn("Build")
 	.Does(() => 
 	{
-		CreateDirectory(PACKAGE_DIR);
-
-        NuGetPack(NUSPEC_PATH, new NuGetPackSettings()
-        {
-            Version = packageVersion,
-            BasePath = BIN_DIR,
-            OutputDirectory = PACKAGE_DIR
-        });
+        NuGetPack("nuget/nunit-project-loader.nuspec",
+			new NuGetPackSettings()
+			{
+				Version = nugetVersion ?? packageVersion,
+				OutputDirectory = OUTPUT_DIR,
+				Files = new [] {
+					new NuSpecContent { Source = "../LICENSE.txt" },
+					new NuSpecContent { Source = binaries + "nunit-project-loader.dll", Target = "tools" }
+				}
+			});
+	});
+
+Task("PackageChocolatey")
+	.IsDependentOn("Build")
+	.Does(() =>
+	{
+		ChocolateyPack("choco/nunit-project-loader.choco.nuspec",
+			new ChocolateyPackSettings()
+			{
+				Version = chocoVersion ?? packageVersion,
+				OutputDirectory = OUTPUT_DIR,
+				Files = new [] {
+					new ChocolateyNuSpecContent { Source = "../LICENSE.txt", Target = "tools" },
+					new ChocolateyNuSpecContent { Source = "VERIFICATION.txt", Target = "tools" },
+					new ChocolateyNuSpecContent { Source = binaries + "nunit-project-loader.dll", Target = "tools" }
```

Comment:
Note: everything should go into the tools directory for chocolatey, because they don't copy files in the root when install the package.